dist/ShareButton.js

```diff
@@ -2086,6 +1921,6 @@ String.prototype.capFLetter = function () {
 exports["default"] = ShareUtils;
 module.exports = exports["default"];
 
-},{}]},{},[22])
-(22)
+},{}]},{},[36])
+(36)
 });
```

Comment:
considering this is in the build, I don't think I can add a newline to the end without postprocessing